pkg/zonemgr/zone_file.go

```diff
@@ -132,17 +131,10 @@ func (zoneFile *ZoneFile) updateVMIRecords(vmiName string, vmiNamespace string,
 		}
 	}
 
-	fmt.Println("********** zoneFile.vmiRecordsMap: ", zoneFile.vmiRecordsMap)
-
 	if isUpdated {
 		zoneFile.updateContent()
 	}
-	return isUpdated, nil
-}
-
-type ARecord struct {
-	value string
-	ip    string
+	return isUpdated
 }
 
 func isIfaceListChanged(ifacesMap map[string]ARecord, interfaces []v1.VirtualMachineInstanceNetworkInterface) (isChanged bool) {
```

Comment:
Please notice that an interface may have more than one IP (one IPv4 and one IPv6). Currently since we support IPv4 only you shoul read the IPv4 value from the `IPs` list.
The `iface.IP` you currently use may have IPv6 value in case of IPv6 only cluster for example.

Reply:
Right good call, just note please that the `IPs` list might be mixed, so need to filter from there the ipv4 one

Reply:
fixed